tests-integration/tests/macros_select.rs

```diff
@@ -4,6 +4,7 @@ use futures::channel::oneshot;
 use futures::executor::block_on;
 use std::thread;
 
+#[cfg_attr(target_os = "wasi", ignore = "WASI: std::thread::spawn not supported")]
```

Comment:
I'm OK w/ this for now, but maybe we can open a tracking issue to remove this? The test itself doesn't *look* like it needs  thread spawning. I'm not exactly sure why it is the way it is.